### PR TITLE
tests: add .coveragerc file

### PR DESCRIPTION
**Pull request by jacquerie**
A simple PR full of nitpickery: see the branch name.

First commit fixes licenses, docstrings and imports of top level files. Second PR adds a `.coveragerc` file ignoring those app instantiation files which are never executed by the application itself (slightly improving coverage).

### CODE REVIEW

File: inspirehep/__init__.py [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
 
-"""inspirehep."""
+"""INSPIREHEP."""
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import, division, print_function
```
Comment (david-caro) [inline inspirehep/__init__.py]: is this intended?

File: inspirehep/version.py [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
 
 """Version information for inspirehep.
 
 This file is imported by ``inspirehep.__init__``,
 and parsed by ``setup.py``.
 """
 
+from __future__ import absolute_import, division, print_function
```
Comment (david-caro) [inline inspirehep/version.py]: This also looks unrelated

File: inspirehep/wsgi.py [hunk 3]
```diff
@@ -1,14 +1,33 @@
...
+"""INSPIREHEP WSGI app instantiation."""
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import, division, print_function
 
 from inspirehep.factory import create_app
 
-application = create_app()
 
+application = create_app()
```
Comment (david-caro) [inline inspirehep/wsgi.py]: And this

File: inspirehep/__init__.py (hunk 1 above)
Comment (jacquerie) [inline inspirehep/__init__.py]: It's a bit useless, but I try to have these future imports everywhere for Python 3 compatibility. Now, it's not doing anything here, but I prefer to have them so people have less chances of copy pasting the wrong boilerplate.

File: inspirehep/wsgi.py (hunk 3 above)
Comment (jacquerie) [inline inspirehep/wsgi.py]: More consistent spacing, I believe.

File: inspirehep/version.py (hunk 2 above)
Comment (jacquerie) [inline inspirehep/version.py]: Ditto.

File: inspirehep/wsgi.py (hunk 3 above)
Comment (david-caro) [inline inspirehep/wsgi.py]: Aaaahhh, it was just moved down... what a misleading diff :)

**Comment by david-caro** [comment]
:+1:

**Comment by david-caro** [comment]
Hmm, not sure that was clear enough. LGTM :+1: :shipit: :bee: :tada: :)
